test/unit/translators/twitter_translators/user_translator_test.rb

```diff
@@ -1,18 +1,18 @@
-require "test_helper"
+require 'test_helper'
 
 class TwitterTranslatorTest < ActiveSupport::TestCase
   def build_twitter_user
     OpenStruct.new(
-      id: "my_twitter_user_id",
-      name: "my_twitter_user_name",
-      email: "my_twitter_user_email@example.org",
-      screen_name: "my_twitter_screen_name",
+      id: 'my_twitter_user_id',
+      name: 'my_twitter_user_name',
+      email: 'my_twitter_user_email@example.org',
+      screen_name: 'my_twitter_screen_name',
       profile_image_uri_https: Faker::Internet.url,
       created_at: 1.hour.ago
     )
   end
 
-  test "creates user when new user" do
+  test 'creates user when new user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.